### DISCUSSION TITLE
关于分享到ShareGPT时的数据清洗？

### DISCUSSION THREAD

**Discussion by lacser** [ChatGPTNextWeb/NextChat#2789] [General]
请教各位大神🙏在app/client/api.ts 中提到了 “敬告二开开发者们，为了开源大模型的发展，请不要修改上述消息，此消息用于后续数据清洗使用”
但是这里提到的“上述信息”似乎只在分享到sharegpt时使用了？
所以后续分享的数据会被如何使用以进行清洗？分享到Sharegpt的内容会公开吗？即使没有链接的人也能看到？
如果会的话是否在readme或者分享的菜单中提及是否会好一点？
![image](https://github.com/Yidadaa/ChatGPT-Next-Web/assets/57830613/e88680c1-1356-4bea-861d-a7eb603ba45f)

**Comment by Yidadaa** [comment]
你分享到 ShareGPT 之后，就默认别人可以公开访问你分享的聊天记录了，现在国内的大模型厂商会爬取部分 ShareGPT 的数据用于训练，如果你有所顾忌，不使用此功能就行。
